.github/workflows/test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     needs: lint
     strategy:
       matrix:
-        ckan-version: [2.9, 2.9-py2, 2.8, 2.7]
```

Comment:
The keitaroinc and qld-gov-au forks are still successfully running tests on 2.8.

Reply:
thanks, I ported some of your changes to support the 2.8 tests